test/src/test-helpers.ts

```diff
@@ -0,0 +1,103 @@
+import * as childProcess from "child_process";
+import * as cps from "current-processes";
+
+export async function closeDesktopApplication(application: string): Promise<boolean> {
+    return new Promise<boolean>(async function (resolve, reject) {
+        let processName: string = "";
+        switch (application.toLowerCase()) {
```

Comment:
It might be good to move this to the Office-Addin-Scripts office-addin-test-helpers package to provide a function to `getProcessNameForApplication(application: string): string`. No change here needed now, but simply a comment as I am reviewing. If you agree, you could add an issue for it in Office-Addin-Scripts.